packs/feats/discordant-voice.json

```diff
@@ -28,7 +28,13 @@
             "remaster": true,
             "title": "Pathfinder Player Core"
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "self:feat:discordant-voice"
```

Comment:
Hmm. Not sure I like adding a self:feat bit. Maybe something other than feat there.

Reply:
I think that's maybe better